packages/kbn-handlebars/scripts/check_for_upstream_updates.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -e
+
+TMP=.tmp-handlebars
+HASH_FILE=packages/kbn-handlebars/src/spec/.upstream_git_hash
+
+# Try to detect Windows environment (I've not tested this!)
+if [[ "$OSTYPE" == "msys" ]]; then
+  # Windows environment
+  DEVNULL=NUL
+else
+  # Everything else (including Cygwin on Windows)
+  DEVNULL=/dev/null
+fi
+
+function cleanup {
+  rm -fr $TMP
+}
+
+trap cleanup EXIT
+
+rm -fr $TMP
+mkdir $TMP
+
+echo "Cloning handlebars repo..."
+git clone -q --depth 1 https://github.com/handlebars-lang/handlebars.js.git -b 4.x $TMP
+
+echo "Looking for updates..."
+hash=`git -C $TMP rev-parse HEAD`
+expected_hash=`cat $HASH_FILE`
```

Comment:
__optional nit:__ I'm not a shell-expert, but my IDE shell script checker doesn't like backticks and offers another solution with this justification: https://github.com/koalaman/shellcheck/wiki/SC2006. Feel free to ignore.

```suggestion
hash=$(git -C $TMP rev-parse HEAD)
expected_hash=$(cat $HASH_FILE)
```